### PR TITLE
Static jsi18n

### PR DESCRIPTION
**Pull request by phlax**
Add static jsi18n app and configure

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ django-overextends==0.4.1
 django-redis==4.4.3
 django-rq>=0.8.0,<=0.9.1
 django-transaction-hooks==0.2
+django-statici18n
```
Comment (unho) [inline requirements/base.txt]: No pinned version? Or at least a minimum version?

**Comment by unho** [comment]
Does this PR finally allows to drop https://github.com/translate/pootle/blob/master/pootle/urls.py#L16-L19 ?

**Comment by phlax** [comment]
@unho good catch - yes it will - when i get it working 8/

**Comment by dwaynebailey** [comment]
Landed in df94bac0ce41875b618f2af10bb6b30394d90365..a60c627be285ca7f6cab545ff89188ff0e92cea3

**Comment by julen** [comment]
Is there a reason why after running `compilejsi18n` the files are located at `STATIC_ROOT` and not at `STATICFILES_DIRS` as expected? Also, any ideas why running this with `USE_I18N = False` creates files for all languages?

**Comment by julen** [comment]
As a heads-up, I have opened #4990 to fix the problems I got with this PR being merged to master.

**Pull request closed without merging**